taxonomy/models.py

```diff
@@ -1042,3 +1042,43 @@ class Meta:
         app_label = 'taxonomy'
         verbose_name = 'Industry'
         verbose_name_plural = 'Industries'
+
+
+class B2CJobAllowList(models.Model):
+    """
+    Model for storing admin configuration for B2C Job Allowlist entries
+
+    .. no_pii:
+    """
+    external_id = models.CharField(
+        max_length=255,
+        blank=False,
+        null=False,
+        unique=True,
+        help_text=_(
+            'The external identifier for the job received from API.'
```

Comment:
nit: Maybe we should add a bit more detail like `EMSI API`?

Reply:
Done